tests/phpunit/test-class-wc-stripe-upe-payment-gateway.php

```diff
@@ -186,11 +203,11 @@ public function test_process_redirect_payment_returns_valid_response() {
 		$payment_method_id = 'pm_mock';
 		$customer_id       = 'cus_mock';
 		$order             = WC_Helper_Order::create_order();
-		$currency          = $order->get_currency();
 		$order_id          = $order->get_id();
 
 		list( $amount, $description, $metadata ) = $this->get_order_details( $order );
 		$order->set_payment_method( WC_Stripe_UPE_Payment_Gateway::ID );
+		$order->save();
```

Comment:
These tests still somehow worked without this line, because the mock gateway also calls `$order->save()`; but honestly, this should be here, because nothing really gets stored to the order unless we save it.